### ISSUE TITLE
rawhide: shadow-utils 4.9 breaks rootless podman containers

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-tracker#916]
The rawhide stream is failing to build because the `ext.config.toolbox` test [is failing](https://jenkins-fedora-coreos.apps.ocp.ci.centos.org/blue/organizations/jenkins/fedora-coreos%2Ffedora-coreos-fedora-coreos-pipeline/detail/fedora-coreos-fedora-coreos-pipeline/2317/pipeline/154) because creating rootless containers is broken:

```
[core@cosa-devsh ~]$ toolbox create --assumeyes
Error: failed to get the Podman version
[core@cosa-devsh ~]$ podman info
Error: cannot setup namespace using newuidmap: exit status 1
```

This was introduced with:

```
shadow-utils 2:4.8.1-20.fc35 -> 2:4.9-1.fc35
```

opened [BZ#1990653](https://bugzilla.redhat.com/show_bug.cgi?id=1990653)

**Comment by dustymabe** [comment]
Should be fixed in `shadow-utils-4.9-2.fc36` ([commit](https://src.fedoraproject.org/rpms/shadow-utils/c/3a328328567ec0e4ec8f6e3dacdba36022b121a2?branch=rawhide))

**Comment by dustymabe** [comment]
Fixed by https://github.com/coreos/fedora-coreos-config/pull/1162
